internal/pkg/model/object.go

```diff
@@ -132,6 +132,7 @@ type Config struct {
 	SharedCode        *SharedCodeConfig      `json:"-" validate:"omitempty,dive" diff:"true"`
 	Orchestration     *Orchestration         `json:"-" validate:"omitempty,dive" diff:"true"`
 	Relations         Relations              `json:"-" validate:"dive" diff:"true"`
+	Metadata          map[string]string      `json:"-" validate:"omitempty,dive" diff:"true"`
```

Comment:
@webrouse potřeboval bych poradit,hromada testů padá na tom, že se tam neočekávaně objevuje prázdná map namísto nil
![CleanShot 2022-01-19 at 16 57 35](https://user-images.githubusercontent.com/215660/150167997-8ee92b21-7968-4dca-9bd4-a4bc989ee9d2.jpg)
Přitom je to tady stejně definovaný jako třeba `Orchestration` která se defaultně v konfigu nezjevuje jestli se dívám správně a ten mapper `Metadata` nastaví jen když je ke konfigu najde (https://github.com/keboola/keboola-as-code/blob/7debb374bf7221639d99cd95b58cd05b31d25672/internal/pkg/mapper/configmetadata/remote.go#L51-L54). V testu taky kontroluju že když je nenajde, tak tu má být `map[string]string(nil)`

Reply:
@JakubMatejka vyzera to na problem s `deepcopy`, ak najde typ `map` tak ho vytvori, aj ked je `nil` ... idem to skusit hned fixnut.

Reply:
https://github.com/keboola/keboola-as-code/pull/457

Reply:
super, děkuju

Reply:
Nebol som si isty, ale nad `nil` `map` sa daju robit operacie, bez toho aby to spadlo, takze je to takto OK.

Povodne som myslel, ci nebudeme potrebovat este nejake metodu `GetMetadata`, ale netreba.

https://go.dev/play/p/89ao8XFUGT-